### PR TITLE
[Automation] - adding v2 chart install test

### PR DESCRIPTION
**Pull request by izaac**
This requires a Rancher and a Downstream cluster with enough resources.

Charts installed: 

- Monitoring v2
- Istio v2
- CIS Benchmark
- Logging

New Optional Environment Variables (with sample values):

```
RANCHER_MONITORING_V2_VERSION=14.5.100
RANCHER_LOGGING_V2_VERSION=3.9.400
RANCHER_KIALI_V2_VERSION=1.32.100
RANCHER_ISTIO_V2_VERSION=1.9.500
RANCHER_CIS_V2_VERSION=1.0.400
TERRAFORM_VERSION=0.12.31 # this has to be this version or newer
RANCHER_REMOVE_V2_CHARTS=False # Optional
```

Existing required Env vars.
```
CATTLE_TEST_URL
ADMIN_TOKEN
USER_TOKEN
RANCHER_CLUSTER_NAME
```

### CODE REVIEW

File: tests/validation/tests/v3_api/test_charts_v2.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    print(tf.apply("--auto-approve"))
+    print("\n\n")
+    check_charts_project_workloads(client, cluster, project_name=project_name)
+
+
+def check_charts_project_workloads(client, cluster, project_name=''):
+    charts_project = client.list_project(name=project_name,
+                                         clusterId=cluster.id).data[0]
+    charts_p_client = get_project_client_for_token(charts_project, USER_TOKEN)
+    for wl in charts_p_client.list_workload().data:
```
Comment (bmdepesa) [inline tests/validation/tests/v3_api/test_charts_v2.py:59]: Can we check that some workloads exist as well?

**Review by bmdepesa — CHANGES_REQUESTED** [review]

**Comment by izaac** [comment]
This will now use the latest version retrieved from the repo using the v1 api.

**Review by bmdepesa — APPROVED** [review]
LGTM

**Comment by izaac** [comment]
This will be closed in favor of the new framework implementation for V2 charts.
There's a V1 charts effort followed up separately.

**Pull request closed without merging**